### PR TITLE
maintaining dependency's 

### PR DESCRIPTION
**Pull request by ProFireDev**

### CODE REVIEW

**Comment by Vukky123** [comment]
mr hotdog, your security policies reference nonexistent "4.0" and "5.0" versions of vukkybot

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```
Comment (nyakase) [inline SECURITY.md]: mr hotdog, your security policies reference nonexistent "4.0" and "5.0" versions of vukkybot

**Review by nyakase — CHANGES_REQUESTED** [review]

File: SECURITY.md (hunk 1 above)
Comment (ProFireDev) [reply SECURITY.md]: forgot that was in there, oops

**Comment by ProFireDev** [comment]
i removed it, ill make a new pr at some point with once that's actually good lol

**Review by nyakase — APPROVED** [review]

**Pull request merged**